tests/PHPStan/Analyser/data/non-empty-string.php

```diff
@@ -206,10 +206,10 @@ public function doFoo4(string $s, array $nonEmptyArrayWithNonEmptyStrings): void
 		assertType('non-empty-string', implode($nonEmptyArrayWithNonEmptyStrings));
 	}
 
-	public function sayHello(): void
+	public function sayHello(int $i): void
```

Comment:
adjusted the existing test, so it still leads to the same result as before this PR.